_data/simple-icons.json

```
@@ -12070,6 +12070,15 @@
             "hex": "EC008C",
             "source": "https://www.veepee.fr/"
         },
+        {
+            "title": "Vega Altair",
+            "hex": "03A9F4",
+            "source": "https://github.com/altair-viz/altair/blob/68d197e8783db5a53a49a246fc2c2ce0f0205ef7/design/altair-logo-light.svg",
+            "license": {
+                "type": "custom",
+                "url": "https://github.com/altair-viz/altair/blob/master/LICENSE"
```

```suggestion
                "url": "https://github.com/altair-viz/altair/blob/68d197e8783db5a53a49a246fc2c2ce0f0205ef7/LICENSE"
```
I'd also version your license, just to play it safe!